source/WorkingPractices/TestSuites/multi-repo_testing.rst

```diff
@@ -0,0 +1,104 @@
+repository is handled can be found :ref:`here <testing>`. Compatible
+code revisions are needed for testing across repositories as described above.
+
+Testing changes in JULES, LFRic Core, UKCA, or any other child repositories is
+as simple as running the standalone test procedures for these codebases.
+
+.. important::
+
+    When specifying an alternative source in the ``dependencies.sh`` file the revision for the source **must** be updated.
+
+    * If setting the source as an fcm URL, the mirror needs to be used and the revision can either be blank (for latest commit) or any valid revision for that branch.
+    * If setting the source as a Working Copy, the hostname needs to be provided (as Hostname:Path) and the revision must be blank.
+
+Testing the UM with other repositories
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To test the UM, any changes to JULES, UKCA, Socrates, CASIM etc will also need
+to be included. This is done by adding another source to the rose stem command
+line.
+
+    1. :ref:`Checkout<checkout>` a UM working copy
+
+        - this may be your branch from a linked ticket, or a clean trunk copy
+          at either the last release or a suitable head of trunk revision.
+
+    2. Run rose stem, including a source code path to every branch involved. As a minimum
+       run ``developer`` group and all groups that cover the repositories being tested.
+
+        .. code-block::
+
+            rose stem --group=developer,jules,ukca --source=. --source=/path/to/jules/changes --source=/path/to/ukca/changes
+
+The source paths involved can either be to local working copies or links to the
+fcm source control e.g. ``fcm:jules.xm_br/dev/user/branch_name``. As many source
+paths as needed can be added to the list.
+
+Testing LFRic Apps with other repositories
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+LFRic Apps testing needs to encompass all of the other repositories affected.
+Paths to the other codebases involved should be added to
+``dependencies.sh`` under each of the ``*_sources`` variables. Again
+these paths can either be to local changes or those in the repository.
+
+    1. :ref:`Checkout<checkout>` an LFRic Apps working copy
+
+        - this may be your branch from a linked ticket, or a clean trunk copy
+          at either the last release or a suitable head of trunk revision.
+
+    2. Update dependencies.sh to point to all other code changes, e.g.
+
+        .. code-block:: RST
+
+            lfric_core_rev=
+            lfric_core_sources=fcm:lfric.xm_br/path/to/branch
+
+            um_rev=
+            um_sources=vldXXX:/path/to/um/working/copy
+
+    3a. Run the lfric_atm developer test-suite
+
+        - suitable for testing changes in other repositories that do not
+          include any LFRic Apps changes
+
+        .. code-block::
+
+            export CYLC_VERSION=8
+            rose stem --group=lfric_atm_developer
+            cylc play <working copy name>
+
```

Comment:
worth adding the cylc gui line to this code block and the one below while we're at it?

Reply:
Done